waterlog/src/Pages/Usage.js

```diff
@@ -19,10 +21,14 @@ class UsageComponent extends Component {
             <Segments/>
             <hr/>
             <h1>Segments Events</h1> 
-            </div>
             <SegmentsEventsList/>
             <hr/>
             <Segmentleaks/>
+            <hr/>
```

Comment:
indent this?

Reply:
how should she indent it it is not nested